pyqt/qtermwidget.sip

```diff
@@ -21,15 +21,14 @@ public:
 
     enum KeyboardCursorShape
     {
-        BlockCursor=0,
-        UnderlineCursor=1,
-        IBeamCursor=2
```

Comment:
Looks like an accident here

Reply:
Oh sure (It's related to #134 , otherwise I cannot build)